templates/zoom-parts/username-section.tpl

```diff
@@ -14,6 +14,12 @@
     <a id="UsernameUsernameList" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/w/index.php?title=Special%3AListUsers&amp;username={$usernamerawunicode|escape:'url'}&amp;group=&amp;limit=1" onMouseUp="$('#UsernameUsernameList').addClass('btn-visited');">Username list</a>
     <a id="UsernameMainspaceSearch" class="btn btn-small" target="_blank" href="https://en.wikipedia.org/w/index.php?title=Special%3ASearch&amp;profile=advanced&amp;search={$usernamerawunicode|escape:'url'}&amp;fulltext=Search&amp;ns0=1&amp;redirs=1&amp;profile=advanced" onMouseUp="$('#UsernameMainspaceSearch').addClass('btn-visited');">Wikipedia mainspace search</a>
     <a id="UsernameGoogleSearch" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=google&amp;data={$usernamerawunicode|escape:'url'}" onMouseUp="$('#UsernameGoogleSearch').addClass('btn-visited');">Google search</a>
+    {if $showinfo}
```

Comment:
I do wonder if this is the right place for this... it'd almost make sense for a new section. Maybe it's something for me to think about when doing the merge to newinternal